--- gnugo-3.4/patterns/mkpat.c~	2003-07-18 20:59:22.000000000 +0200
+++ gnugo-3.4/patterns/mkpat.c	2005-04-11 14:44:58.207120624 +0200
@@ -3206,9 +3206,9 @@
     /* Forward declaration, which autohelpers might need. */
     if (database_type != DB_FULLBOARD) {
       if (database_type != DB_CORNER)
-	fprintf(output_FILE, "extern struct pattern %s[];\n\n", prefix);
+	fprintf(output_FILE, "static struct pattern %s[];\n\n", prefix);
       else
-	fprintf(output_FILE, "extern struct corner_pattern %s[];\n\n", prefix);
+	fprintf(output_FILE, "static struct corner_pattern %s[];\n\n", prefix);
     }
 
     /* Write the autohelper code. */
